playbooks/common/openshift-master/config.yml

```diff
@@ -304,6 +231,30 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
+# Create master certificates separately to speed up configuring
+# masters (since they are configured serially). The
+# openshift_master_certificates role may be added as a dependency of
+# the openshift_master role once masters are configured in parallel.
```

Comment:
Afraid I cannot recall, I worked through a lot of weird issues when I was refining my local setup but I can't find any details on that one, I do remember something vaguely. Sorry!